uwsgi.ini

```diff
@@ -6,7 +6,6 @@
 # https://www.techatbloomberg.com/blog/configuring-uwsgi-production-deployment/
 socket = /tmp/kolibri_uwsgi.sock
 chmod-socket = 660
-gid = www-data
```

Comment:
This has to be re-worked to use `chown-socket` so Nginx can read/write to the socket.